packages/venia-concept/src/components/MiniCart/miniCart.js

```diff
@@ -212,21 +212,24 @@ class MiniCart extends Component {
     };
 
     get miniCartInner() {
-        const { checkout, productList, props } = this;
+        const { checkout, productConfirm, productList, props, state } = this;
         const { classes, isCartEmpty, isMiniCartMaskOpen } = props;
+        const { isEditPanelOpen } = state;
```

Comment:
Where does `isEditPanelOpen` ever get set to `true`?

I think we may be missing some code in this file?

Reply:
I looked carefully where, and how to use `isEditPanelOpen`, and yes, it is always `false`. I will remove all references `isEditPanelOpen`.